src/Amadevus.RecordGenerator.Generators/RecordPartialGenerator.cs

```diff
@@ -112,7 +112,7 @@ MethodDeclarationSyntax CreateRecordMutator(RecordDescriptor.Entry entry)
                     .AddModifiers(SyntaxKind.PublicKeyword)
                     .WithParameters(
                         Parameter(
-                            entry.Identifier)
+                            Identifier(Names.Value))
```

Comment:
That's an important change. Could you note it additionally down in changelog?

Also, just from diff, shouldn't you update `arguments` creation as well? The tests don't pass.

Which also triggered me to check why the AppVeyor succeeded and it turns our AppVeyor discovers 0 tests... Off to fixing that.

Reply:
I think I lost some words on the way from thought to typing: the `value` is not passed into Update call, which results in the current property value being passed instead.

Reply:
Sorry, this was my bad. I was only running the tests from `GeneratedTypesNamingConventionsTests` and forgot to run all the tests before doing my commit and push!

Fixed in d9a3f3e55601bdc475e5469a882a27ddfc1adfbe.

Reply:
> That's an important change. Could you note it additionally down in changelog?

Done in 9ea4d9cee6ed9d6550f81a38c88508f7a85eea3b.